technologies/app/streamlit/metadata.yaml

```diff
@@ -0,0 +1,26 @@
+version: v1
+type: APP
+id: streamlit
+label: Streamlit
+baseline: Streamlit
+description: Streamlit
+available: true
+iconPath: streamlit.png
+backgroundColor: "#3F4F75"
+customFlags: []
+contexts:
+  - id: streamlit-1.31.0
+    label: 1.31.0
+    releaseNotes: First version of Streamlit 1.31.0 into Saagie.
+    available: true
+    trustLevel: stable
+    recommended: true
+    ports:
+      - port: 8501
+        name: Streamlit
+        rewriteUrl: true
+        basePath: SAAGIE_BASE_PATH
+    dockerInfo:
+      image: "glay/streamlit"
```

Comment:
Maybe it's better to put on saagie/streamlit ?

Reply:
Yes but I don't have the rights to do it.
So I pushed the image on my repository